### ISSUE TITLE
Mai'Zoth respawn time is too high

### ISSUE THREAD

**Issue by Contrubius** [azerothcore/azerothcore-wotlk#11301]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [X] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour
Mai'Zoth takes way too long to respawn considering he is an non-elite mob. I had a timer set after I killed him the first time and it took him more than 30 minutes to respawn on PTR:

1. Timer started after I killed him:
![WoWScrnShot_040622_212101](https://user-images.githubusercontent.com/100415626/162072700-565f1789-c5f6-4634-8f6e-f2d44ae34167.jpg)

2. Guards respawn after 9 minutes:

![WoWScrnShot_040622_213020](https://user-images.githubusercontent.com/100415626/162072864-c0283d40-155e-4436-824f-4a61ab979798.jpg)

3. Mai'Zoth spawns back after 31 minutes
![WoWScrnShot_040622_215223](https://user-images.githubusercontent.com/100415626/162073059-1aa0fb7a-3438-4079-a236-5b9c34bd516e.jpg)

### Expected Blizzlike Behaviour

It should take around 5 minutes for Mai'Zoth to respawn considering he is not even an elite mob since patch 2.3 TBC.
### Source

https://classic.wowhead.com/npc=818/maizoth

### Steps to reproduce the problem

1. Go Inside the Mosh'Ogg Ogre Mound Cave in Stranglethorn Vale, Coords for Mai'Zoth are 52, 27
2. Kill Mai'Zoth
3. Wait 30 minutes for him to respawn.

### Extra Notes

Originally reported: https://github.com/chromiecraft/chromiecraft/issues/3267
Reported by: OA350
### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ccb6d3e5a1401df8c9e99a384b9a5eead62441a1

### Operating system

Windows 10.

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Nyeriah** [comment]
The issue is that many mobs should respawn faster if more players are in the zone, it’s particularly the case with many STV mobs.

**Comment by Dejanos** [comment]
> 

Does it have anything to do that maybe most of them were elites in Vanilla and were downgraded to regular mobs in 2.3? All other Ogres in the cave take around 9 minutes to respawn, which is normal, Mai'Zoth who used to be an elite but is a regular mob now takes more than 30 minutes. I looked through issues and saw no report on this particular mob, if there are several similar reports or more reports on STV NPCs I will close this issue so it doesn't trouble anyone.

**Comment by Si1ker** [comment]
the issue is not dynamic respawn as nyeriah suggested.
It actually has 30 min respawn timer, the issue is the lack of source pointing the correct respawn timer (there's none)

(523, 818, 0, 0, 0, 0, 0, 1, 1, 1, -12342.7, -1156.72, 2.48357, 2.25148, **1800**, 0, 0, 2217, 1534, 0, 0, 0, 0, '', 0),
1800sec = 30min timer

**Comment by Nyeriah** [comment]
There is no source pointing to the correct respawn timer and there likely won’t be one because, as I said, their timers are not fixed and vary depending on how many players are in the vicinity.

**Comment by Si1ker** [comment]
Deleted my comment 'cause I don't wanna cause any drama, it has a fixed timer of 30min, is an issue, nothing to do with dynamic respawn, is all I gotta say.

Here's the npc, 1800secs is a fixed 30min timer.
```sql
(523, 818, 0, 0, 0, 0, 0, 1, 1, 1, -12342.7, -1156.72, 2.48357, 2.25148, **1800**, 0, 0, 2217, 1534, 0, 0, 0, 0, '', 0)
```
bye. 🐱

**Comment by Nyeriah** [comment]
I don’t believe there’s drama in disagreeing with someone else’s opinion. I don’t discard the possibility the base respawn timer is wrong. However, from my own experience, mobs don’t have fixed respawn timers. This can be researched on retail with moderate easy.

**Comment by heyitsbench** [comment]
For the record, it seems the mobs in this cave on Wrath Classic (49345) on a 'medium' (realmlist) pop server are much closer to the suggested 5 minute respawn time rather than 30 minutes.

**Comment by Sunnypotato** [comment]
comment from quest [The Mind's eye](https://www.wowhead.com/wotlk/quest=591/the-minds-eye#comments;40)
![image](https://github.com/user-attachments/assets/785afb28-b124-4daf-a78e-3eda0ec9f1df)
commenter pretty much just explaining that the mobs respawned in the order they were previously killed, indicating Mai'zoth's respawn timer seems to be the same as the other mobs. 
I think the respawn timer was shortened from 30min to 5min for all the mobs in this cave when they were nerfed to be non-elite in TBC
